integration/client_sdk/go/utils.go

```diff
@@ -97,3 +123,187 @@ func setupNetwork(channel string) (*gateway.Network, error) {
 
 	return network, nil
 }
+
+func setup(ccID, ccPath string, initEnclave bool) error {
+
+	var enclavePeers []string
+	if initEnclave {
+		enclavePeers = append(enclavePeers, "jdoe_test.sampleorg.example.com")
+	}
+
+	nwDetails := &testutils.NetworkDetails{
+		ChannelID:    "mychannel",
+		Peers:        []string{"jdoe_test.sampleorg.example.com"},
+		Orderers:     []string{"orderer.example.com"},
+		EnclavePeers: enclavePeers,
+	}
+
+	consortium := []string{"SampleOrg"}
+
+	mrenclave, err := fpc.ReadMrenclave(ccPath)
+	if err != nil {
+		return err
+	}
+
+	ccDetails := &testutils.ChaincodeDetails{
+		Id:           ccID,
+		Path:         ccPath,
+		Version:      mrenclave,
+		Lang:         fpc.ChaincodeLang,
+		Policy:       fpcpolicy.MajorityOfMembers(consortium),
```

Comment:
As mentioned on the call, from a blind copy&paste perspective i think this is even more brittle than the old single org policy. So a forward comment would be useful.  A url to the godoc would be even better but i was too lazy to figure that out :-)
```suggestion
		Policy:       fpcpolicy.MajorityOfMembers(consortium), // Note this assumes a static network, see comments on this function
```
```suggestion
		Policy:       fpcpolicy.MajorityOfMembers(consortium),
```

Reply:
oh didn't see this suggestion before I pushed an update with a similar comment :)

Reply:
> oh didn't see this suggestion before I pushed an update with a similar comment :)

my comment hit github probably just shortly before you pushed.  That said, there are two aspects i like about mine: having it on the same line as the function is a bit more robust from copy/paste perspective and having the forward reference to the (more extensive) godoc comment on the function itself.  Arguably, as long as you have a static network and majority is what you want/need this is robust, so we do not really need the "testing" disclaimer?